src/attr.rs

```diff
@@ -1452,6 +1469,17 @@ impl<'a, T: AsRef<[u8]> + ?Sized> Parseable<NlaBuffer<&'a T>> for Nl80211Attr {
                 }
                 Self::SchedScanPlans(nlas)
             }
+            NL80211_ATTR_VENDOR_ID => {
+                Self::VendorId(parse_u32(payload).context(format!(
+                    "Invalid NL80211_ATTR_VENDOR_ID {payload:?}"
+                ))?)
+            }
+            NL80211_ATTR_VENDOR_SUBCMD => {
+                Self::VendorSubcmd(parse_u32(payload).context(format!(
```

Comment:
Other variants didn't seem to be specifying an endianness so I didn't here either

Reply:
It most probably is the same endianess as the requesting machine anyway, so this should work